src/webview/pipeline/app/common/item.ts

```diff
@@ -0,0 +1,54 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+
+import { SelectWidget } from '../element/selectwidget';
+import { EditItem } from '../element/maincontent';
+import { createDiv } from '../utils/util';
+import { InputWidget } from '../element/inputwidget';
+import { ButtonsPanel } from '../element/buttonspanel';
+import { PipelineStart, Trigger } from './types';
+import { disableButton } from '..';
+
+export function disableRemoveButton(event: Node & ParentNode): void {
```

Comment:
I think that should be member of the `Button` class

Reply:
I am using this function in two different files, I will move this function in a different file. I will create disablebutton file and move the function disableRemoveButton and disableSelection in that file.